Touch Bar Simulator/TouchBarWindow.swift

```diff
@@ -295,6 +300,17 @@ final class TouchBarWindow: NSPanel {
 	func removeTitlebar() {
 		styleMask.remove(.titled)
 	}
+	
+	func roundCorners(_ corners: CACornerMask) {
+		self.contentView?.layer?.cornerRadius = 7.5
+		self.contentView?.layer?.maskedCorners = []
+
+		self.contentView?.layer?.maskedCorners = corners
+		self.contentView?.layer?.borderWidth = 0.0
```

Comment:
This logic could be moved out into a `CALayer` extension to be reusable.

Could also add a `NSWindow` extension that used this and also handled `isOpaque`, etc.